ompi/mca/osc/osc.h

```diff
@@ -152,7 +152,80 @@ typedef int (*ompi_osc_base_component_select_fn_t)(struct ompi_win_t *win,
                                                    struct opal_info_t *info,
                                                    int flavor,
                                                    int *model);
+/**
+ * OSC nonblocking component select
```

Comment:
I'm not sure this is needed - the "select" is choosing which module to use for supporting the RMA functionality. It is already a local (nonblocking) action, I think. So I'm pretty sure we don't have to split it into a begin piece and a complete piece.